tests/failpoints/cases/test_snap.rs

```diff
@@ -243,6 +243,58 @@ fn test_destroy_peer_on_pending_snapshot() {
     must_get_equal(&cluster.get_engine(3), b"k120", b"v1");
 }
 
+#[test]
+fn test_destroy_peer_on_abort_snapshot() {
+    let mut cluster = new_server_cluster(0, 3);
+    configure_for_snapshot(&mut cluster);
+    let pd_client = Arc::clone(&cluster.pd_client);
+    pd_client.disable_default_operator();
+
+    let r1 = cluster.run_conf_change();
+    pd_client.must_add_peer(r1, new_peer(2, 2));
+    pd_client.must_add_peer(r1, new_peer(3, 1003));
+
+    cluster.must_put(b"k1", b"v1");
+    // Ensure peer 3 is initialized.
+    must_get_equal(&cluster.get_engine(3), b"k1", b"v1");
+
+    cluster.must_transfer_leader(1, new_peer(1, 1));
+
+    cluster.add_send_filter(IsolationFilterFactory::new(3));
+
+    for i in 0..20 {
+        cluster.must_put(format!("k1{}", i).as_bytes(), b"v1");
+    }
+
+    let apply_snapshot_fp = "region_apply_snap_abort";
+    fail::cfg(apply_snapshot_fp, "return()").unwrap();
```

Comment:
Now that failpoint is used to trigger abort, then destroy is unnecessary. Only need to do is waiting an election timeout.

Reply:
You're right. In fact here there's no destroy triggered in the test case. The failpoint before_peer_destroy_1003_fp is just to make sure peer_destroy won't run in any case so that we can wait the tick run in snapshot abort status.

Reply:
then the failpoint `before_peer_destroy_1003_fp` is unnecessary.